specs/windows/nt_info.table

```diff
@@ -0,0 +1,23 @@
+table_name("nt_info") description(
```

Comment:
The formatting  in spec file looks a bing inconsistent with other specs. May I ask you to fix it?